### PR TITLE
migrate minSpan

### PR DESCRIPTION
**Pull request by mtanda**
related to https://github.com/grafana/grafana/pull/10891

### CODE REVIEW

File: public/app/features/dashboard/dashboard_migration.ts [hunk 1]
```diff
@@ -363,6 +363,11 @@ export class DashboardMigrator {
 
     if (oldVersion < 16) {
       this.upgradeToGridLayout(old);
+      panelUpgrades.push(function(panel) {
```
Comment (torkelo) [inline public/app/features/dashboard/dashboard_migration.ts]: should this not be inside the upgradeToGridLayout ?

Comment (mtanda) [reply public/app/features/dashboard/dashboard_migration.ts]: need to pass panelUpgrades to upgradeToGridLayout(), is it okay?

Comment (mtanda) [reply public/app/features/dashboard/dashboard_migration.ts]: I first write there, but test fails.
I can't understand why it fails, and then I think the place is not correct, I should use panelUpgrades.

**Comment by mtanda** [comment]
test fails...

**Comment by mtanda** [comment]
@torkelo I fix it.

**Comment by torkelo** [comment]
thanks @mtanda!

**Pull request merged**